spec/controllers/api/users_controller_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe API::UsersController, type: :controller do
+  let(:user) { classroom_teacher }
+
+  describe "GET #authenticated_user", :vcr do
+    before do
+      get :authenticated_user, params: {
+        access_token: user.api_token
+      }
+    end
+
+    it "returns success" do
```

Comment:
I think its worth adding a failure case too 🛑

Reply:
Couldn't really think of any failure cases here other than the user not being authorized which is already tested as I mentioned above